Readme.md

```diff
@@ -36,6 +48,51 @@ fastify.get('/two', (req, reply) => {
 })
 ```
 
+### Remote Cache Store
+
+`fastify-caching` offers the connectivity to a remote store like shown below with `ioredis` and `abstract-cache`.
+See `fastify-caching` [documentation](https://github.com/fastify/fastify-caching)
+
+
+```js
+// This example requires the following packages to be installed
+// - ioredis
+// - abstract-cache
+
+const IORedis = require('ioredis')
+const redis = new IORedis({host: '127.0.0.1'})
+const abcache = require('abstract-cache')({
+  useAwait: false,
```

Comment:
```suggestion
  useAwait: true,
```

Sorry. This actually a mistake in the upstream documentation that I haven't gotten around to fixing. The `abstract-cache-redis` module implements the `await` protocol.